kafka/api.html

```diff
@@ -3871,23 +3871,24 @@ <h3 id="NewProducer">
      conf is a *ConfigMap with standard librdkafka configuration properties.
     </p>
     <p>
-     Supported special configuration properties:
+     Supported special configuration properties (type, default):
     </p>
     <pre>go.batch.producer (bool, false) - EXPERIMENTAL: Enable batch producer (for increased performance).
                                   These batches do not relate to Kafka message batches in any way.
                                   Note: timestamps and headers are not supported with this interface.
 go.delivery.reports (bool, true) - Forward per-message delivery reports to the
                                    Events() channel.
-go.delivery.report.fields (string, all) - Comma separated list of fields to enable for delivery reports.
-                                          Allowed values: all, none (or empty string), key, value
+go.delivery.report.fields (string, "key,value") - Comma separated list of fields to enable for delivery reports.
+                                    Allowed values: all, none (or empty string), key, value, headers
+                                    Warning: There is a performance penalty to include headers in the delivery report.
 go.events.channel.size (int, 1000000) - Events().
 go.produce.channel.size (int, 1000000) - ProduceChannel() buffer size (in number of messages)
 go.logs.channel.enable (bool, false) - Forward log to Logs() channel.
 go.logs.channel (chan kafka.LogEvent, nil) - Forward logs to application-provided channel instead of Logs(). Requires go.logs.channel.enable=true.
 </pre>
     <h3 id="Producer.AbortTransaction">
      func (*Producer)
-     <a href="//golang.org/src/github.com/confluentinc/confluent-kafka-go/kafka/producer.go?s=30574:30636#L900">
+     <a href="//golang.org/src/github.com/confluentinc/confluent-kafka-go/kafka/producer.go?s=30707:30769#L900">
```

Comment:
May I ask the meaning of the number here? "30707:30769"

Reply:
Not really sure, but guessing they're links to byte positions in the source file.